azkaban-webserver/src/main/java/azkaban/webapp/servlet/ScheduleServlet.java

```diff
@@ -684,6 +689,87 @@ private void ajaxScheduleFlow(HttpServletRequest req,
     ret.put("message", projectName + "." + flowName + " scheduled.");
   }
 
+  /**
+   *
+   * This method is in charge of doing cron scheduling.
+   * @throws ServletException
+   */
+  private void ajaxScheduleCronFlow(HttpServletRequest req,
```

Comment:
This is a reasonably large piece in itself. Do you think this should be in a separate class of its own ?